app/assets/stylesheets/tripshow.scss

```diff
@@ -1,44 +1,81 @@
-* {
-  margin: 0;
-  padding: 0;
-}
-.trip-show-main {
-  h1 {
+@import "variables"; 
+.tripshow-container{
+    color: $blue-purple;
+    padding: 4.375rem;
     text-align: center;
+  h1{
+    font-family: $header;
```

Comment:
I'm seeing ```h1 { font-family: $header; }``` appear many times in different style sheets. If h1 always needs to be styled the same way, the h1 style should live in the _upper_ most stylesheet so that it's applied globally.. Maybe consider application.scss for global styles?

Reply:
head i fixed that im starting to see som ereapeating styles that i want us mob program together